types/src/transaction/deploy.rs

```diff
@@ -1008,6 +1008,44 @@ impl Deploy {
         )
     }
 
+    /// Creates an add bid deploy, for testing.
+    #[cfg(any(all(feature = "std", feature = "testing"), test))]
+    pub fn add_bid(
+        chain_name: String,
+        auction_contract_hash: AddressableEntityHash,
+        public_key: PublicKey,
+        amount: U512,
+        delegation_rate: u8,
+        timestamp: Timestamp,
+        ttl: TimeDiff,
+    ) -> Self {
+        let payment = ExecutableDeployItem::ModuleBytes {
+            module_bytes: Bytes::new(),
+            args: runtime_args! { ARG_AMOUNT => U512::from(3_000_000_000_u64) },
```

Comment:
This is not critical but we might want to set this to default min delegation dynamically (presumably coming from whatever the Default for a chainspec is) in the future